packages/devtools_app/lib/src/memory/flutter/memory_graph_model.dart

```diff
@@ -222,6 +260,11 @@ class HeapGraph {
   /// Group all instances by all classes - with applied filters.
   final Map<String, List<HeapGraphElementLive>> groupByClass = {};
 
+  /// Group all filtered out instances by all classes (removed from groupByClass).
+  final Map<String, List<HeapGraphElementLive>> filteredElements = {};
```

Comment:
it could be handy to comment on what the String key for each of this maps is.

Reply:
Good idea, done.

```
  /// Group all classes by libraries (key is library, value are classes).
  /// This is the entire set of objects (no filter applied).
  final Map<String, Set<HeapGraphClassLive>> rawGroupByLibrary = {};

  /// Group all classes by libraries (key is library, value is classes).
  /// Filtering out objects that match a given filter. This is always a
  /// subset of rawGroupByLibrary.
  final Map<String, Set<HeapGraphClassLive>> groupByLibrary = {};

  /// Group all instances by class (key is class name, value are class
  /// instances).  This is the entire set of objects (no filter applied).
  final Map<String, Set<HeapGraphElementLive>> rawGroupByClass = {};

  /// Group all instances by class (key is class name, value are class
  /// instances).  Filtering out objects that match a given filter. This
  /// is always a subset of rawGroupByClass.
  final Map<String, Set<HeapGraphElementLive>> groupByClass = {};

  /// Group of instances by filtered out classes (key is class name, value
  /// are instances). These are the instances not in groupByClass, together
  /// filteredElements and groupByClass are equivalent to rawGroupByClass.
  final Map<String, Set<HeapGraphElementLive>> filteredElements = {};

  /// Group of libraries by filtered out classes (key is library name, value
  /// are classes). These are the libraries not in groupByLibrary, together
  /// filteredLibraries and groupByLibrary are equivalent to rawGroupByLibrary.
  final Map<String, Set<HeapGraphClassLive>> filteredLibraries = {};
```